### DISCUSSION TITLE
"Chia cannot be closed"

### DISCUSSION THREAD

**Discussion by punx4life85** [Chia-Network/chia-blockchain#15042] [Support]
Trying to update to 1.7.1 on win 11, was win 10 that I recently updated.. keep getting the error that "chia cannot be closed. Please close it manually and click retry to continue." Uninstalled everything, have restarted multiple times, looked through task manager for processes still running, cleared app data folders.. nadda, what am I missing??
![dbec4a7f-bad8-41dc-9168-1b5602772904](https://user-images.githubusercontent.com/130851625/232233125-9dab8067-a5e8-4146-b8e7-fe92e7eb485d.jpg)

**Comment by malahhaor** [comment]
+1 same issue.
